src/operator/quantization/quantized_conv.cc

```diff
@@ -86,12 +89,10 @@ bool QuantizedConvType(const nnvm::NodeAttrs& attrs,
   const ConvolutionParam& param = nnvm::get<ConvolutionParam>(attrs.parsed);
   CHECK_EQ(in_type->size(), param.no_bias? 6U : 9U);
   CHECK_EQ(out_type->size(), 3U);
-  TYPE_ASSIGN_CHECK(*in_type, 0, mshadow::kInt8);
```

Comment:
?

Reply:
Since mkldnn support u8 input data type while cudnn use s8, so we move input data type check into FComputeEx.

Reply:
If you moved the code to FComputeEx, is the type inference complete? Can you use
```cpp
#if MXNET_USE_MKLDNN == 1
```
here?

Reply:
OK, will add MXNET_USE_MKLDNN check here